algorithm.ipynb

```diff
@@ -120,6 +120,16 @@
     "len(languages)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_placeholders = spark.read.parquet(\"image_placeholders\")\n",
```

Comment:
How about we make the parquet files path a notebook parameter, that defaults to `image_placeholder`? Just so we can centralise things later on.